src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -204,31 +162,31 @@ class ValidationStep extends React.Component {
                         </View>
                         <View style={[styles.mv5, styles.flex1]}>
                             <TextInput
+                                inputID="amount1"
+                                shouldSaveDraft
+                                defaultValue=""
```

Comment:
No need to pass an empty default value. We should check if amount exists in [filterInput](https://github.com/Expensify/App/blob/23ebd43804fc930cd640ef49a7fa874f8928d0bd/src/pages/ReimbursementAccount/ValidationStep.js#L87), i.e. `let value = amount ? amount.trim() : '';`